api/src/main/java/org/pmiops/workbench/cdr/model/Criteria.java

```diff
@@ -0,0 +1,235 @@
+    }
+
+    @Column(name = "name")
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Criteria name(String name) {
+        this.name = name;
+        return this;
+    }
+
+    @Column(name = "is_group")
+    public boolean getGroup() {
+        return group;
+    }
+
+    public void setGroup(boolean group) {
+        this.group = group;
+    }
+
+    public Criteria group(boolean group) {
+        this.group = group;
+        return this;
+    }
+
+    @Column(name = "is_selectable")
+    public boolean getSelectable() {
+        return selectable;
+    }
+
+    public void setSelectable(boolean selectable) {
+        this.selectable = selectable;
+    }
+
+    public Criteria selectable(boolean selectable) {
+        this.selectable = selectable;
+        return this;
+    }
+
+    @Column(name = "est_count")
+    public String getCount() {
+        return count;
+    }
+
+    public void setCount(String count) {
+        this.count = count;
+    }
+
+    public Criteria count(String count) {
+        this.count = count;
+        return this;
+    }
+
+    @Column(name = "concept_id")
+    public String getConceptId() {
+        return conceptId;
+    }
+
+    public void setConceptId(String conceptId) {
+        this.conceptId = conceptId;
+    }
+
+    public Criteria conceptId(String conceptId) {
+        this.conceptId = conceptId;
+        return this;
+    }
+
+    @Column(name = "domain_id")
+    public String getDomainId() {
+        return domainId;
+    }
+
+    public void setDomainId(String domainId) {
+        this.domainId = domainId;
+    }
+
+    public Criteria domainId(String domainId) {
+        this.domainId = domainId;
+        return this;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Criteria criteria = (Criteria) o;
+
+        if (id != criteria.id) return false;
+        if (sortOrder != criteria.sortOrder) return false;
+        if (parentId != criteria.parentId) return false;
+        if (group != criteria.group) return false;
+        if (selectable != criteria.selectable) return false;
+        if (type != null ? !type.equals(criteria.type) : criteria.type != null) return false;
+        if (code != null ? !code.equals(criteria.code) : criteria.code != null) return false;
+        if (name != null ? !name.equals(criteria.name) : criteria.name != null) return false;
+        if (count != null ? !count.equals(criteria.count) : criteria.count != null) return false;
+        if (conceptId != null ? !conceptId.equals(criteria.conceptId) : criteria.conceptId != null) return false;
+        return domainId != null ? domainId.equals(criteria.domainId) : criteria.domainId == null;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = (int) (id ^ (id >>> 32));
```

Comment:
You can use Objects.hash() here:

https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#hash(java.lang.Object...)

Reply:
ok